docker/package/tezos_setup_wizard.py

```diff
@@ -98,9 +98,9 @@ def is_full_snapshot(import_mode):
     id="network",
```

Comment:
A few lines above here there is the definition of the welcome message which says:
```
In order to run a baking instance, you'll need the following Tezos binaries:
 tezos-client, tezos-node, tezos-baker-<proto>.
If you have installed tezos-baking, these binaries are already installed.
```
which should be corrected, however instead of renaming these I think we should actually correct that you don't need those binaries installed, but rather the packages (because this wizard depends on the systemd services and the rest of the package too).

So, I suggest changing it to just:
```
In order to run a baking instance, you'll need the following Tezos packages:
 tezos-client, tezos-node, tezos-baker-<proto>.
If you have installed tezos-baking, these packages are already installed.
```